03.asynchronous/callback.js

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env node
+
+import sqlite3 from "sqlite3";
+import timers from "timers/promises";
+
+const database = new sqlite3.Database(":memory:");
+
+// エラー無し
+database.run(
+  "CREATE TABLE books (id INTEGER PRIMARY KEY AUTOINCREMENT, title TEXT NOT NULL UNIQUE)",
+  () => {
+    database.run(
+      "INSERT INTO books (title) VALUES(?)",
```

Comment:
`VALUES` の後ろにスペースを入れたほうがよいと思います。

```suggestion
      "INSERT INTO books (title) VALUES (?)",
```

Reply:
単語の区切りはスペースを入れるようにします！
他の文字列内のSQL分もチェックします。

Reply:
https://github.com/Ryo-ta090972/js-practices/pull/4/commits/9ac1d4100e2b09932c61e06a4c2f12b688fb8d15 で対応済みです。